### ISSUE TITLE
Change tracker on Firefox and Jquery > 3.1.1

### ISSUE THREAD

**Issue by abbottmw** [loopindex/ckeditor-track-changes#150]
I am viewing the Demo of the change tracker in Firefox, and the OpenTip display is off.  Text appears vertical.  Looking at the debugger, it shows the width of that node as only 1px.     When I downgraded my jquery to 3.1.1, it displayed correctly.  

I know the demo code is creating a script to jquery 1.9.1, but our apps out there are using the latest jQuery, and this causes an issue only in Firefox.

**Comment by imdfl** [comment]
Hi @abbottmw 
In the commercial version of the change tracker, we've given up on the opentip library. If we use a different tooltip control which happens to be open source and reasonably compatible with the LITE code, I'll post here the details (and in the unlikely event that I have some spare time, I'll even integrate it in a branch).

**Comment by abbottmw** [comment]
Yeah, I was going through the opentip library and downloaded the latest and trying to see if updating the  the adapter code in LITE would work.   I could get the tooltip to show up in Firefox, but not the message and it wasnt in the right spot, so slowly trying to tweak it.
